default/buildings.txt

```diff
@@ -999,6 +1026,7 @@ BuildingType
             Planet environment = Adequate
         ]
     ]
+    EnqueueLocation = [[ENQUEUE_BUILD_ONE_PER_PLANET]]
```

Comment:
OK for that too. What is the best way to remove the three lines : discard the PR and create a new one or add a corrective PR on top of it?